index.html

```diff
@@ -212,10 +212,24 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital signatures or proofs of existence, for bounded documents such as verifiable credentials.
+This specification defines how to express proofs of integrity, such as digital
+signatures or proofs of existence, for bounded documents such as verifiable
```

Comment:
```suggestion
signatures or proofs of existence, for bounded documents, such as verifiable
```
Could leave this comma out, IFF removing the one after `proofs of integrity`.